include/boost/json/basic_parser_impl.hpp

```diff
@@ -686,10 +686,17 @@ parse_value(const char* p,
             // or just leave it as is
             if(BOOST_JSON_UNLIKELY(p == sentinel()))
                 return maybe_suspend(p, state::val2);
-            // intentional fallthrough
+            BOOST_FALLTHROUGH;
+
         case ' ':
+            BOOST_FALLTHROUGH;
```

Comment:
This and the next few are not necessary. It's a common idiom to group cases like this. I don't believe any compiler would complain.

Reply:
GCC doesn't seem to issue a warning there even with the explicit flag. To be honest, marking every empty case like this looks very ugly, so I'm going to bite the bullet and insist you remove those lines.